ci/azure/macos_script

```diff
@@ -62,11 +62,25 @@ stage3-release/bin/zig build test docs \
   --search-prefix "$PREFIX"
 
 if [ "${BUILD_REASON}" != "PullRequest" ]; then
-  mv ../LICENSE stage3-release/
-  mv ../zig-cache/langref.html stage3-release/
+  # Produce the experimental std lib documentation.
+  mkdir -p stage3-release/docs/std
+  $ZIG test ../lib/std/std.zig \
+    --zig-lib-dir ../lib \
+    -femit-docs=stage3-release/docs/std \
+    -fno-emit-bin
```

Comment:
Please move this to above the if statement so that a regression is revealed in a pull request before it makes it to master branch.